nostr-verify.php

```diff
@@ -20,7 +20,7 @@
  * Add rewrite rules
  */
 function generate_rewrite_rules() {
-	add_rewrite_rule( '^.well-known/nostr.json', 'index.php?well-known=nostr', 'top' );
+	add_rewrite_rule( '^\\.well-known/nostr\\.json', 'index.php?well-known=nostr', 'top' );
```

Comment:
Why the double encoding here? Could we do

```suggestion
	add_rewrite_rule( '^\.well-known\/nostr\.json', 'index.php?well-known=nostr', 'top' );
```

Reply:
> Why the double encoding here?

Because we are not escaping a dot in this string but send a backslash and a dot to add_rewrite_rule.

Reply:
I don't really understand what you mean here. Could you expand on that a bit?

Reply:
The first parameter of add_rewrite_rule is a regular expression.
It begins with `^\.` literal string.
In PHP syntax we should write `'^\\.'` to get the literal string `^\.`

Although entering `'^\.'` produces the same value, it not safe.
Mostly because it bends one's mind to believe that one could enter anything and it will remain the same.

Reply:
You may close my PR-s
as I think and write in long-term **only** (2+ years).
An I have to refactor human errors as stated above.

Reply:
Ah okay. That makes sense. I suppose it won't work. How about this additional change then?

```suggestion
	add_rewrite_rule( '^\\.well-known\/nostr\\.json', 'index.php?well-known=nostr', 'top' );
```

Reply:
Doesn't core escape regexp delimiters??? 😲